### DISCUSSION TITLE
Dashboard: Anzeige Verbindungsstatus Fahrzeug zu Wallbox auch ohne SOC Fahrzeug möglich?

### DISCUSSION THREAD

**Discussion by WolfImBusch** [solectrus/solectrus#3653] [Q&A] [answered]
Hallo liebe Community,
ich habe gerade - leider mal wieder - das Problem, dass ich den SOC meine Fahrzeugs nicht auslesen kann, da sich die API zu dem Hersteller Backend geändert hat.

Ist es im Dashboard dennoch möglich, den Verbindungsstatus des Fahrzeugs zu der Wallbox (also über die Umgebungsvariable INFLUX_SENSOR_WALLBOX_CAR_CONNECTED) anzuzeigen oder benötigt das Dashboard für eine korrekte Anzeige stets auch die Umgebungsvariable INFLUX_SENSOR_CAR_BATTERY_SOC? 

Ich vermute letzteres .. :-) Ich sammle den Verbindungsstatus aktuell über MQTT ein und schreibe auch die Werte in die InfluxDB - leider zeigt sich aktuell das Dashboard davon unbeeindruckt, obwohl ich in der compose.yml im Abschnitt "Dashboard -> environment" nämlich "INFLUX_SENSOR_WALLBOX_CAR_CONNECTED" eingetragen habe und sich die UI nach Neustart des Dashboard nicht geändert hat.

Herzlichen Dank für einen Tipp und viele Grüße

**Comment by ledermann** [comment, marked as answer]
Derzeitig ist es so, dass die Darstellung mit Auto-Ladestand und Hausakku-Ladestand nur erfolgt, wenn **beide** Sensoren (`CAR_BATTERY_SOC` und `BATTERY_SOC`) definiert sind **und** auch einen Wert haben. Angezeigt wird dann auch der Connect-Status. Andernfalls wird nur der Hausakku dargestellt.

Hintergrund ist, dass die Darstellung seltsam aussehen würde, wenn neben dem Auto-Icon gar nichts steht.

Du könntest aber tricksen: Wenn du die beiden Sensoren definierst und per MQTT irgendwie dafür sorgst, dass der Ladestand des Autos `0` ist (also nicht `NULL`), dann wird es so angezeigt, wie von dir gewünscht.
